### ISSUE TITLE
Two little problems about From and TableView

### ISSUE THREAD

**Issue by HUSTlwr** [Meragon/Unity-WinForms#29]
First one, I can only adjust  the spacing of the cells by CellPadding, but I can't just adjust the spacing between rows and keep the column spacing the same. I didn't find relevant attribute values in the TableView class.
Second one, I want to use my own picture as the close icon in the upper right corner of the form. My picture looks like this: 
![image](https://user-images.githubusercontent.com/50614735/74033637-92610800-49f1-11ea-9a97-b972fae1d97b.png)

But, after I replaced the close.png in the Resources folder with my own picture, the close icon looks like this in the unity:
![c76b31cb45ff9b5b98ddd7053d2d9c8](https://user-images.githubusercontent.com/50614735/74033405-0222c300-49f1-11ea-8075-e0a5f1a37db2.png)
I wonder why the background color of my picture will be changed.

thanks!

**Comment by Meragon** [comment]
Yeah, TableView control don't support lot of stuff and still kinda buggy.
As for second issue, you need to use transparent image with with white color, while you use black color on gray background.

**Comment by HUSTlwr** [comment]
I have a little suggestion about the close button in the upper right corner of the form. Hover image of close button should be added to the class AppGdiImages and class AppResources. Changing image would be easier than modifying color of close button  when the mouse is hovering over the button.

**Comment by Meragon** [comment]
There is a property 'uwfImageHover' in Button class, you can change this.
`public class MyForm : Form {
    public MyForm() {
        CloseButton.uwfImageHover = this.uwfAppOwner.Resources.Checked;
    }
}`

**Comment by HUSTlwr** [comment]
Yes, I have found and used this property. However, what I mean is the hover image of close button should be added to the class AppGdiImages and class AppResources as image resource, just like Close imge and Checked image. Only by this way, can we import our image to UnityWinForms class on Inspector. I did, and I modified the '_MakeButtonClose() ' method in the Form class.
`private void _MakeButtonClose()`
`{`
 `...`
` closeButton.uwfBorderColor = Color.Transparent;`
`closeButton.uwfBorderHoverColor = Color.Transparent;`
`//closeButton.uwfHoverColor = Color.FromArgb(64, 252, 252, 252);`
`//closeButton.uwfImageColor = Color.FromArgb(64, 64, 64);`
`//closeButton.uwfImageHoverColor = Color.FromArgb(128, 128, 128);`
`closeButton.uwfImageHover = uwfAppOwner.Resources.CloseHover;`
`...`
`}`
Anyway, your work helps me a lot. Respect.
